### PR TITLE
Test IDs added and working for Create Account Flow

### PR DESCRIPTION
**Pull request by M4RcB4**
It should be pretty self explanatory. Let me know what kind of information you would like for these PRs that add test IDs. They will be pretty common and hopefully we can streamline their integration into the code as much as possible. We will be waiting to merge the tests that depend on them until these IDs are available in master to avoid having failing tests from IDs.

### CODE REVIEW

File: packages/edge-login-ui-rn/src/components/abSpecific/AccountInfoComponent.js [hunk 1]
```diff
@@ -149,6 +149,7 @@ class AccountInfoComponent extends React.Component<Props, State> {
     return (
       <View style={style.top}>
         <TextAndIconButton
+          testID={this.props.testID}
```
Comment (swansontec) [inline packages/edge-login-ui-rn/src/components/abSpecific/AccountInfoComponent.js:152]: This was not added to the props above. Flow should have caught this in the pre-commit hook.

File: packages/edge-login-ui-rn/src/components/common/Checkbox.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import * as React from 'react'
 import { Image, Text, TouchableWithoutFeedback, View } from 'react-native'
 
 type Props = {
+  testID?: String,
```
Comment (swansontec) [inline packages/edge-login-ui-rn/src/components/common/Checkbox.js:7]: Lowercase `string`, also caught by Flow.

File: packages/edge-login-ui-rn/src/components/common/MaterialInput.js [hunk 3]
```diff
@@ -83,6 +84,7 @@ export class MaterialInput extends React.Component<Props, State> {
         : this.props.autoCorrect
     return (
       <TextField
+        testID={this.props.testID}
```
Comment (swansontec) [inline packages/edge-login-ui-rn/src/components/common/MaterialInput.js:87]: Error. You will need to add `testID` to the file `flow-typed/react-native-material-textfield.js`

**Review by swansontec — CHANGES_REQUESTED** [review]
Found some Flow issues.

**Comment by M4RcB4** [comment]
@swansontec I squashed the commits. Should be ready for approval/merge. Do we wait for re-review, or do you usually merge it if the changes requested were added?

**Review by swansontec — APPROVED** [review]

**Pull request merged**